test/axisLabel.html

```diff
@@ -267,7 +268,7 @@
                 option = {
                     backgroundColor: '#eee',
                     title: {
-                        text: 'axisLable.formatter: {value} should be correct'
+                        text: 'axisLabel.formatter: {value} should be correct'
```

Comment:
Related test cases should be put into `timeScale-formatter.html`

Reply:
related test case has been move into timeScale-formaatter.html